server/url_prefixer.go

```diff
@@ -80,9 +80,9 @@ func (up *URLPrefixer) ServeHTTP(rw http.ResponseWriter, r *http.Request) {
 	// extract the flusher for flushing chunks
 	flusher, ok := rw.(http.Flusher)
 	if !ok {
-		up.Logger.
-			WithField("component", "prefixer").
-			Fatal("Expected http.ResponseWriter to be an http.Flusher, but wasn't")
+		msg := "Expected http.ResponseWriter to be an http.Flusher, but wasn't"
```

Comment:
This was kind of meant as a sanity check, which is why the error is Go-specific and logged. If we return this to the user, maybe we should try to front-load some of the Googling work and figure out why a `http.ResponseWriter` might not be a `http.Flusher`.